README.md

```diff
@@ -101,49 +102,45 @@ library in `build/Release/lib`. The headers will be in `build/Release/include`.
 A debug library can be created by adding `-DCMAKE_BUILD_TYPE=Debug` to the cmake command above.  The -j3 could be used 
 with the `make install` command to use multiple threads.
 
-### Simple Source Installation On Windows (MS Visual Studio 2017) 
-
-This is a work in progress so not everything works quite yet.
+### Simple Build On Windows (MS Visual Studio 2017) 
```

Comment:
Is this duplicate of the Using graphical solution below? Maybe just add a link to that, as the section describes using IDEs

Reply:
The using IDE's section is general info for any 'other' IDE and does not apply to VS.  Actually the most useful information for setting up other IDE's would be the description of how to build from the command line.